_sass/components/_identifier.scss

```diff
@@ -0,0 +1,140 @@
+.contact-bar {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.usa-identifier {
+  background-color: $theme-identifier-background-color;
+  color: $theme-identifier-color;
+  padding-top: 1rem;
+  padding-bottom: 3rem;
+
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.5rem;
+
+  a {
+    color: $theme-identifier-color;
+  }
+
+  ul, ol {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+}
+
+.usa-identifier__required-links-list a {
+  font-size: 1.5rem;
+}
+
+a.usa-identifier__required-link {
+  color: $theme-identifier-color;
+
+	 &:hover {
+     color: $theme-identifier-font-color-secondary;
+   }
+}
+
+.usa-identifier__container {
+  @media #{$breakpoint-lg} {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.usa-identifier__logos {
+	padding-top: 1rem;
+  padding-right: 2rem;
+
+	@media #{$breakpoint-lg} {
+    padding-right: 2rem;
+	}
+}
+
+.usa-identifier__logo {
+  width: 5rem;
+  display: block;
+}
+
+
+.usa-identifier__identity-disclaimer {
+  font-size: 1.6rem;
+}
+
+.usa-identifier__identity-disclaimer a,
+.usa-link--external,
+a.usa-identifier__identity-disclaimer-link {
+  font-weight: 700;
+}
+
+.usa-identifier__identity-disclaimer-link a,
+.usa-identifier__identity-disclaimer a:visited {
+  color: $theme-identifier-font-color-secondary;
+}
+
+a.usa-identifier__identity-disclaimer-link:hover,
+.identifier__identity-disclaimer a:hover {
+  color: #f0f0f0;
+}
+
+.usa-identifier__identity {
+  padding-top: 1rem;
+
+	@media #{$breakpoint-lg} {
+		padding-top: 0;
+	}
+
+	p {
+		padding: 0;
+		line-height: 1;
+		margin: .5rem 0 0 0;
```

Comment:
Is reseting the right, bottom, and left margins needed vs `padding-top: 0.5rem`?

Reply:
Yes, it's needed to keep the text spacing closer to resemble the 3.0 identifier layout.